### ISSUE TITLE
Add always on top support

### ISSUE THREAD

**Issue by gyj1109** [Yubico/yubioath-desktop#558]
- Yubico Authenticator version: 5.0.2
- Operating system and version: Windows 10 1909
- YubiKey model and version: N/A
- Bug description summary: N/A

I wonder if it's possible to add an option to always show the window on top so that I could see the code while inputting? Or at least only make it on Windows (I don't use Mac and don't very sure if macOS support "always on top")? Thank you very much!

**Comment by dagheyman** [comment]
Thanks for the suggestion! Just a question, are you aware you can copy the code to the clipboard by double-clicking or using the copy icon in the top bar?

**Comment by gyj1109** [comment]
> Thanks for the suggestion! Just a question, are you aware you can copy the code to the clipboard by double-clicking or using the copy icon in the top bar?

Yeah, I know I could copy it very easy, but some websites don't allow you to paste the code because they just give you six boxes. It will automatically move the focus backwards, but this cannot be pasted easily.

**Comment by dainnilsson** [comment]
Yubico Authenticator 6.0 has now been released and uses a new codebase. As such, this issue has been marked with the `legacy` label, and will be automatically closed in 7 days. If this issue is still relevant to Yubico Authenticator 6, please comment on the issue saying so, and it will be kept open (or be re-opened). Sorry for the inconvenience!
